src/bindings/capi/thorvg_capi.h

```diff
@@ -1336,23 +1330,24 @@ TVG_API Tvg_Result tvg_shape_get_stroke_gradient(const Tvg_Paint* paint, Tvg_Gra
 *
 * \note To reset the stroke dash pattern, pass @c nullptr to @p dashPattern and zero to @p cnt.
 */
-TVG_API Tvg_Result tvg_shape_set_stroke_dash(Tvg_Paint* paint, const float* dashPattern, uint32_t cnt);
+TVG_API Tvg_Result tvg_shape_set_stroke_dash(Tvg_Paint* paint, const float* dashPattern, uint32_t cnt, float offset);
 
 
 /*!
-* \brief Gets the dash pattern of the stroke.
```

Comment:
@hermet I added BETA_API - not sure whether it's necessary.

Reply:
If you think it's ready to official release then you don't need to add BETA_API  tag. Otherwise you should.

Reply:
Ok, now this breaks the api compatibility.